frontend/components/command.tsx

```diff
@@ -0,0 +1,16 @@
+export const Command = ({ text }: { text: string }) => (
+  <div className="flex flex-col sm:flex-row items-baseline justify-between">
+    <div className="mb-4 flex overflow-auto">
+      <span className="mr-4 select-none">$</span>
+      <pre>
+        <code>{text}</code>
+      </pre>
+    </div>
+    <button
+      onClick={() => navigator.clipboard.writeText(text)}
+      className="px-2 py-1 font-heading font-medium rounded-full hover:bg-gray text-black transition active:bg-dark-gray active:text-white"
```

Comment:
I think removing the overflow scroll caused an issue on mobile:

<img width="504" alt="image" src="https://user-images.githubusercontent.com/667029/170081539-83097f67-23a5-4672-b381-bc7a436ed037.png">

maybe we can add it only for small viewports?

and maybe we should hide the copy button on mobile, what do you think?

Reply:
Here is what it looks like after the latest update: 
![image](https://user-images.githubusercontent.com/29112081/170166900-38e628d3-7c03-4249-835b-89f015cb7fbe.png)